todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskActivity.java

```diff
@@ -50,6 +50,9 @@ protected void onCreate(Bundle savedInstanceState) {
         AddEditTaskFragment addEditTaskFragment =
                 (AddEditTaskFragment) getSupportFragmentManager().findFragmentById(R.id.contentFrame);
 
+        // Prevent the presenter from loading data from the repository if this is a config change.
+        boolean shouldLoadDataFromRepo = addEditTaskFragment == null;
```

Comment:
This is very cryptic, i'd rather prefer something like this:
boolean doNotLoadDataOnConfigChange = false;
if {null == aETF} { doNotLoadDataOnConfigChange = true}

This saves you the comment and the reader immediately knows what's going on